trb2024/docker-compose.yml

```diff
@@ -340,7 +340,7 @@ services:
       - /etc/localtime:/etc/localtime:ro
       - /etc/timezone:/etc/timezone:ro
   econolite-virtual-controller:
-    image: usdotfhwastol/econolite-virtual-controller:latest
+    image: usdotfhwastoldev/econolite-virtual-controller:latest
```

Comment:
The docker build script also changes the tag into candidate. Maybe the fix is to allow some custom exception for this image's automated build until we can provide public image.

Reply:
EDIT: after the evc repository and its docker build scripts now created. we can have instructions to have the user build with `-d` so that the name automatically gets tagged ` usdotfhwastoldev/econolite-virtual-controller:develop`. 
So only last change required here is to rename the `latest` tag into `develop`

Reply:
Oh also let's also apply the change to xil_cloud and xil_cloud_telematics as well while we are at it.